scripts/ci/images/ci_wait_for_all_prod_images.sh

```diff
@@ -15,6 +15,16 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+if [[ ${WAIT_FOR_IMAGE} != "true" ]]; then
+    # shellcheck source=scripts/in_container/_in_container_script_init.sh
+    . "$( dirname "${BASH_SOURCE[0]}" )/../libraries/_script_init.sh"
+    echo
+    echo "Not waiting for all PROD images to appear as they are built locally in this build"
+    echo
+    push_pull_remove_images::determine_github_registry
```

Comment:
I don't understand why we need this -- the `GITHUB_REGISTRY` env variable is already set globaly in the Workflow.

Reply:
Yeah. I figured that we can do it this way when I saw how you've done the self-hosted selection. It was not clear before that '' value was 'falsy' in the expression language. It makes the whole workflow logic much simpler. I was not hurrying with that as INFRA is not in a hurry with providing a token for us (only today Gavin assigned the ticket to himself, so I hope it will happen soon).